store/postgres/src/dynds/shared.rs

```diff
@@ -63,17 +63,16 @@ pub(super) fn load(
     for (vid, name, context, address, creation_block) in dds.into_iter() {
         if address.len() != 20 {
             return Err(constraint_violation!(
-            "Data source address 0x`{:?}` for dynamic data source {} in deployment {} should have be 20 bytes long but is {} bytes long",
-            address, vid, id,
-            address.len()
-        ));
+                "Data source address `0x{:?}` for dynamic data source {} should be 20 bytes long but is {} bytes long",
+                address, vid,
+                address.len()
+            ));
         }
-        let address = Some(H160::from_slice(address.as_slice()));
 
         let creation_block = creation_block.to_i32();
         let data_source = StoredDynamicDataSource {
             name,
-            address,
+            params: Some(json_from_bytes(&address)),
```

Comment:
`json_from_bytes` is infallible, as it is _encoding_ to base64.

Reply:
I am a bit confused about what's going on with the change from `address` to `params`

Reply:
The params received in [dataSource.create](https://github.com/graphprotocol/graph-ts/blob/main/common/datasource.ts#L7) are now chain-agnostic, and in the private ds table will be stored as jsonb. Each chain can choose how to format this json, and the format is expressed here in `as_stored_dynamic_data_source`.

For Ethereum, I'm proposing we store the address as a base64 encoded string, since that's the closest thing to a byte array that json gives us. For the purpose of this PR we are not actually storing base64 yet, since private dses are yet to be implemented, but just having `StoredDynamicDataSource` handle a json value in this format.